python/py-upt-rubygems/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-upt-rubygems
+version             0.2
+revision            0
+
+platforms           darwin
+supported_archs     noarch
+license             BSD
+maintainers         {@korusuke somaiya.edu:karan.sheth} openmaintainer
+
+description         RubyGems frontend for upt
+long_description    ${description}
+
+homepage            https://framagit.org/upt/upt-rubygems
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+checksums           sha256  5e71e20a43d0a6b263dc703a080df77e89ffd1cf3ce1a984f261958041c93406 \
+                    rmd160  1c3eaf3080ade1e5d4225aecd39c3b0eedbd1e64 \
+                    size    5180
+
+python.versions     37
```

Comment:
since we're now not using it as a module anymore, I think you will need to use ```python.default_version  37``` instead.

Reply:
Isn't this comment only relevant for upt itself?

Reply:
oh, yeah sure - sorry... please ignore me ;)